cli/cmd/list.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/list"
+	"github.com/tarantool/tt/cli/modules"
+)
+
+// NewListCmd creates list command.
+func NewListCmd() *cobra.Command {
+	var listCmd = &cobra.Command{
+		Use:   "list",
```

Comment:
Shouldn't it be a sub-command of `tt list`. Soon we will have a new functionality for installed binaries listing. Will this be `tt list` or other command? For example it can be `tt list instances` and `tt list binaries` and more in future. @LeonidVas what do you think?

Reply:
Made it tt list instances.
Maybe tt list apps will be better?